### PR TITLE
Add excludes for CLJS compiler warnings

### PR DESCRIPTION
**Pull request by Hendekagon**
the CLJS compiler was giving the following warnings:
WARNING: Use of undeclared Var cljs.core/time at line 802 out/garden/selectors.cljc
WARNING: time already refers to: cljs.core/time being replaced by: garden.selectors/time at line 802 out/garden/selectors.cljc
WARNING: link at line 802 is being replaced at line 852 out/garden/selectors.cljc
WARNING: rem already refers to: cljs.core/rem being replaced by: garden.units/rem at line 242 out/garden/units.cljc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Garden
 
+WARNING: Use of undeclared Var cljs.core/time at line 802 out/garden/selectors.cljc
```
Comment (jeluard) [inline README.md]: Are those supposed to be there?

File: project.clj [hunk 2]
```diff
@@ -1,4 +1,4 @@
-(defproject garden "1.3.1-SNAPSHOT"
+(defproject hendekagon/garden "1.3.1-SNAPSHOT"
```
Comment (jeluard) [inline project.clj]: I don't think we want to merge this change.

File: src/garden/selectors.cljc [hunk 3]
```diff
@@ -738,7 +738,7 @@
     label
     legend
     li
-    link
+    ;link
```
Comment (jeluard) [inline src/garden/selectors.cljc]: Is that intended?

File: src/garden/units.cljc [hunk 4]
```diff
@@ -10,6 +10,8 @@
    [garden.util :as util]
    #?(:cljs
       [cljs.reader :refer [read-string]]))
+  #?(:cljs
+     (:refer-clojure :exclude [rem]))
```
Comment (jeluard) [inline src/garden/units.cljc]: The same `exclude` exists for `clj` so it's probably cleaner to merge both.

File: project.clj (hunk 2 above)
Comment (Hendekagon) [inline project.clj]: oops no

File: src/garden/selectors.cljc (hunk 3 above)
Comment (Hendekagon) [inline src/garden/selectors.cljc]: to address  WARNING: link at line 802 is being replaced at line 852 out/garden/selectors.cljc

File: src/garden/units.cljc (hunk 4 above)
Comment (Hendekagon) [inline src/garden/units.cljc]: k

File: README.md (hunk 1 above)
Comment (Hendekagon) [inline README.md]: no - removed

File: src/garden/selectors.cljc (hunk 3 above)
Comment (jeluard) [inline src/garden/selectors.cljc]: I don't see why this warning is generated. Can we let this change out of this PR and address it separately?
It's not quite the same as other warnings and I would like to make sure we properly fix it.

Comment (Hendekagon) [inline src/garden/selectors.cljc]: ok

**Comment by jeluard** [comment]
Let me know when you made the changes and I will merge this PR.

**Comment by Hendekagon** [comment]
please check the code now cheers

**Comment by jeluard** [comment]
Looking at it further I can see the same issues have already been fixed recently. You can see the relevant merge [here](https://github.com/noprompt/garden/pull/85/files).
Apparently your clone was not up-to-date. Sorry for not noticing this before.

**Comment by Hendekagon** [comment]
ok no worries...but I only cloned it yesterday so I don't know how I didn't get that merge

**Comment by jeluard** [comment]
Weird. Not sure why that would happen.

**Pull request closed without merging**